packages/flutter/test/widgets/image_test.dart

```diff
@@ -370,6 +398,20 @@ class TestImageProvider extends ImageProvider<TestImageProvider> {
   String toString() => '${describeIdentity(this)}()';
 }
 
+class TestImageStreamCompleter extends ImageStreamCompleter {
```

Comment:
this can probably be a one line mockito too maybe

Reply:
I think with mockito the assertion here will be more complex (e.g instead of asserting zero listeners registered, you need to assert that evert call to addListener is followed by a call to removeListener)